README.md

```diff
@@ -91,6 +91,30 @@ handles a multitude of input configuration types:
 In the absence of this flag, a default configuration is loaded from `.semgrep.yml`
 or multiple files matching `.semgrep/**/*.yml`.
 
+### Registry
+
+As mentioned above, you may also specify a `registry_name` as configuration.
+[r2c](https://r2c.dev) provides a [registry](https://github.com/returntocorp/semgrep-rules)
```

Comment:
Thoughts on leading with the semgrep.live/registry page instead of the repository?

Reply:
Oh, I thought I did this already actually. Must've been in a different README.